pages/common/irssi.md

```diff
@@ -0,0 +1,27 @@
+# irssi
+
+> Text based IRC client since 1999. See also: https://irssi.org for full documentation.
+
+- Open irssi and connect to a server with a nickname:
+
+`irssi -n {{nickname}} -c {{server.name.com}}`
+
+- Open irssi and connect with a specific nickname, server and port:
```

Comment:
You can remove the `-n` from this example since its already shown in the first one. We can focus on the port more by saying - `Open irssi and connect to a server at a given port`.